src/api/common/index.js

```diff
@@ -9,7 +9,7 @@ module.exports = {
   dropsToXrp: utils.dropsToXrp,
   xrpToDrops: utils.xrpToDrops,
   toRippledAmount: utils.toRippledAmount,
-  generateWallet: utils.generateWallet,
+  generateAddress: utils.generateAddress,
```

Comment:
It is because you get used to it.

Words `RippleName` and `RippleAddress` used all over documentation, and casual developer will expect to find same in code.



On Mon, Aug 17, 2015 at 8:00 PM -0700, "sublimator" <notifications@github.com> wrote:
> @@ -9,7 +9,7 @@ module.exports = {
>    dropsToXrp: utils.dropsToXrp,
>    xrpToDrops: utils.xrpToDrops,
>    toRippledAmount: utils.toRippledAmount,
> -  generateWallet: utils.generateWallet,
> +  generateAddress: utils.generateAddress,

I think 'ripple' is kinda implied?


---
Reply to this email directly or view it on GitHub:
https://github.com/ripple/ripple-lib/pull/509/files#r37260251